client/src/components/VideoRecorder.js

```diff
@@ -67,6 +73,7 @@ const VideoRecorder = ({ questionId, jobId, userId, onTimerActiveChange }) => {
         userId,
         jobId,
         questionId,
+        interviewId,
```

Comment:
Same with this. Do we need interviewId when we have both userId and jobId?

Reply:
In future if we keep the number of attempts then we need to change the code and as well as we need to change the files on the azure storage also. Having the interview Id won't affect anything now and as well as in future we can maintain that file structure for a long time.